build.gradle

```diff
@@ -460,7 +461,7 @@ dependencies {
     runtimeOnly 'com.fasterxml.woodstox:woodstox-core:6.4.0'
     runtimeOnly 'org.apache.ws.xmlschema:xmlschema-core:2.2.5'
     runtimeOnly 'org.apache.santuario:xmlsec:2.2.3'
-    runtimeOnly 'com.github.luben:zstd-jni:1.5.2-1'
+    runtimeOnly "com.github.luben:zstd-jni:${versions.zstd}"
```

Comment:
@cwperks I haven't checked by do you happen to know why ZSTD is needed for security plugin? thank you.

Reply:
Hey @reta, I think the one is related to the core changes about the [snapshot compression](https://github.com/opensearch-project/OpenSearch/pull/2996). I think we dont rely on this for any part of our code, but we do have this related to the `:bundlePlugin` in `build.gradle`.

Reply:
I'm not entirely sure, but it looks related to kafka.

This runtimeOnly dependency was added in the switch to the opensearch standard build.gradle file here: https://github.com/opensearch-project/security/commit/03a224d16045a8f561e2d7d84b8765c95309524c

And it was upgraded once when kafka was upgraded: https://github.com/opensearch-project/security/pull/2484

Kafka is one of the available sinks for audit logging.

I am taking a deeper look into what those runtimeOnly dependencies are in the build.gradle file.

Reply:
Thank you folks

Reply:
I believe its utilized here at runtime: https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/auditlog/sink/KafkaSink.java#L75-L85

There's more details on the KafkaProducer from IBM here: https://developer.ibm.com/articles/benefits-compression-kafka-messaging/